### PR TITLE
FIX: via mesh

### PR DESCRIPTION
**Pull request by gkorompi**

### CODE REVIEW

**Comment by ring630** [comment]
Customer can still run pyedb on older aedt version. Why do you want to remove the version check?

**Comment by gkorompi** [comment]
@ring630 please check comment here: 
https://github.com/ansys/pyedb/issues/352

**Review by ring630 — CHANGES_REQUESTED** [review]
via_mesh_plating should not be allowed in 23.2 and older version

**Comment by gkorompi** [comment]
It will not crash, it depends solely on the AEDT version that is used to open the design. If you define the variable in 23.2 edb and you launch it with 24.2 aedt, then the option will be there. Also, the dataset is not updated until saved, so it makes no sense to check it as soon as it is not updated! @svandenb-dev  what's your opinion?

File: src/pyedb/dotnet/edb_core/edb_data/hfss_simulation_setup_data.py [hunk 1]
```diff
@@ -977,19 +977,14 @@ def via_mesh_plating(self):
         -------
         bool
         """
-        if self._parent._pedb.version[0] >= 10:
-            return self._via_settings.ViaMeshPlating
-        else:
-            self._parent._pedb.logger.error("Property only supported on Ansys release 2024R1 and later")
-            return False
+        return self._via_settings.ViaMeshPlating
+        self._parent._pedb.logger.warning("Property only supported on Ansys release 2024R1 and later")
```
Comment (SMoraisAnsys) [inline src/pyedb/dotnet/edb_core/edb_data/hfss_simulation_setup_data.py:981]: Just so you know, this log warning will never be triggered as a return if performed just before.

**Comment by ring630** [comment]
@gkorompi I am not convinced by this PR. I closed it for now.

**Comment by ring630** [comment]
@gkorompi #515

**Pull request closed without merging**